src/views/limit_order_list_view.js

```diff
@@ -0,0 +1,146 @@
+import Backbone from 'backbone';
+import LimitOrder from '../models/limit_order';
+import TradeView from './trade_view';
+import QuoteListView from './quote_list_view';
+import LimitOrderList from '../collections/limit_order_list';
+import LimitOrderView from './limit_order_view';
+
+const LimitOrderListView = Backbone.View.extend({
+  initialize(params){
+    console.log(params);
+    console.log('these are the params');
+    this.template = params.template;
+    this.hamRadio = params.hamRadio;
+    this.dropdownTemplate = params.dropdownTemplate;
+    this.listenTo(this.hamRadio, 'render_order_dropdown', this.renderOrderDropdown);
+    this.listenTo(this, 'order_purchase', this.addLimitOrder);
+    this.listenTo(this, 'order_sell', this.addLimitOrder);
```

Comment:
As far as I can tell, you don't have any code to trigger `order_purchase` or `order_sell` events.